cmd/kubeapps-apis/plugins/kapp_controller/packages/v1alpha1/server_test.go

```diff
@@ -6067,7 +6153,7 @@ func TestGetInstalledPackageResourceRefs(t *testing.T) {
 					},
 					ObjectMeta: metav1.ObjectMeta{
 						Namespace: "default",
-						Name:      "my-installation-ctrl",
+						Name:      "my-installation.apps.k14s.io",
```

Comment:
This is the actual change in Kapp 0.47. However, it is still an opt-in feature, so even if using kapp 0.47, users will still have the `-ctrl` thing.